apps/docs/docs/template/api/experimental/_category_.json

```
@@ -0,0 +1,8 @@
+{
+  "label": "Experimental API",
+  "position": 4,
+  "link": {
+    "type": "generated-index",
+    "description": "Experimental API reference for @rx-angular/template"
+  }
```

issue: Remove the index page for consistency.
```suggestion
  "link": null
```

---

Can be resolved without commit according to https://github.com/rx-angular/rx-angular/pull/1436#issuecomment-1278902812.